core/services/ardupilot_manager/frontend/index.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>ArduPilot Manager</title>
+    <link rel="stylesheet" href="/static/css/metro-all.css" />
+    <link rel="stylesheet" href="/style.css" />
+    <script src="/static/js/vue.js"></script>
+    <script src="/static/js/axios.min.js"></script>
+    <script src="/static/js/metro.js"></script>
+</head>
+
+<body>
+    <div id="app">
+        <div class="container">
+            <h1>ArduPilot Manager</h1>
+            <h3>Endpoints:</h3>
+
+            <div class="grid">
+                <div class="row p-1 m-1 bg-gray">
+                    <div class="cell-2">
+                        <div>Name</div>
```

Comment:
It appears that if name is over 9 characters it does not accept it anymore, showing the reason of why an endpoint could not be created is really important since it's logic sometimes what is happening for a human user.

Reply:
Really? Didn't notice that.
It should accept from 2 till 50 characters on both name and owner.
Will check that.